style.css

```diff
@@ -0,0 +1,60 @@
+* {
+  margin: 0;
+  padding: 0;
+  font-family: playfair Display, sans-serif;
+}
```

Comment:
- [ ] Original design is font is not that. You should use this font  [Poppins].(https://fonts.google.com/specimen/Poppins) You can  check this [article](https://developer.mozilla.org/en-US/docs/Learn/CSS/Styling_text/Web_fonts) for how to use google fonts.


**Your design:**

![image](https://user-images.githubusercontent.com/38635158/125689374-07c20ede-015a-4023-b153-86bb7ec64ebe.png)
___
**Original design:**

![image](https://user-images.githubusercontent.com/38635158/125689473-6f17bd65-a25b-4861-9602-f9e7826199f8.png)